docs/wallets/mainnet-address-generation.md

```diff
@@ -13,12 +13,12 @@ Crypto.org Chain has [registered](https://github.com/satoshilabs/slips/blob/mast
 Coin Type: 394 <br />
 Derivation Path: `44'/394'/0'/0/{index}`  where index starts from 0.
 
-i.e. the first address is derived at path `44'/394'/0'/0/0` and second one at path `44'/394'/0'/0/1`.
+i.e. the first address is derived at path `44'/394'/0'/0/0` and the second one at path `44'/394'/0'/0/1`.
 
 ## How to generate address
 
-::: warning MAKE SURE YOU BACKUP YOUR MNEMONIC WORDS
-Before you use the generated mainnet addresses to receive funds on mainnet, make sure that you have backup your address's mnemonic words in a safe place and the mnemonic words are correct such that it can restore to your wallet address.
+::: warning: MAKE SURE YOU BACKUP YOUR MNEMONIC WORDS
+Before you use the generated mainnet addresses to receive funds on mainnet, make sure that you have backup your address's mnemonic words correctly in a safe place and the mnemonic words are correct such that it can restore to you wallet address in future.
```

Comment:
```suggestion
Before you use the generated mainnet addresses to receive funds on mainnet, make sure that you have backup your address's mnemonic words correctly in a safe place and the mnemonic words are correct such that it can restore to your wallet address in the future.
```